sql-cli/sql_cli/configuration.py

```diff
@@ -78,3 +78,25 @@ def write_value_to_yaml(self, section: str, key: str, value: str) -> None:
         filepath = self.get_filepath()
         with open(filepath, "w") as fp:
             yaml.dump(yaml_config, fp)
+
+    def write_config_to_yaml(self) -> None:
+        """
+        Write a particular key/value to the desired configuration.yaml.
+
+        Example:
+        ```
+            [section]
+            - item1
+            - item2
+        ```
+
+        :param section: Section within the YAML file where the key/value will be recorded
+        :param values: List of items to be written to the YAML file
+        """
+        yaml_config = self.from_yaml_to_dict()
+        yaml_config["connections"] = self.connections
+        yaml_config["airflow_home"] = self.airflow_home
+        yaml_config["airflow_dags_folder"] = self.airflow_dags_folder
```

Comment:
We probably want to always record these values in the config - but this can be done in a separate PR